class-sc-wp-updates-notifier.php

```diff
@@ -743,6 +875,60 @@ public function sc_wpun_settings_main_field_hide_updates() {
 			</select>
 			<?php
 		}
+
+		/** Email settings **/
+		public function sc_wpun_settings_email_notifications_text() {
+		}
+
+		public function sc_wpun_settings_email_notifications_field_email_notifications() {
+			$options = $this->get_set_options( self::OPT_FIELD );
+			?>
+			<label><input name="<?php echo esc_attr( self::OPT_FIELD ); ?>[email_notifications]" type="checkbox" value="1" <?php checked( $options['email_notifications'], 1 ); ?> /> <?php esc_html_e( 'Yes', 'wp-updates-notifier' ); ?>
+			</label>
+			<?php
+		}
+
+		public function sc_wpun_settings_email_notifications_field_notify_to() {
+			$options = $this->get_set_options( self::OPT_FIELD );
+			?>
+			<input id="sc_wpun_settings_email_notifications_notify_to" class="regular-text" name="<?php echo esc_attr( self::OPT_FIELD ); ?>[notify_to]" value="<?php echo esc_attr( $options['notify_to'] ); ?>" />
+			<span class="description"><?php esc_html_e( 'Separate multiple email address with a comma (,)', 'wp-updates-notifier' ); ?></span>
+			<?php
+		}
+
+		public function sc_wpun_settings_email_notifications_field_notify_from() {
+			$options = $this->get_set_options( self::OPT_FIELD );
+			?>
+			<input id="sc_wpun_settings_email_notifications_notify_from" class="regular-text" name="<?php echo esc_attr( self::OPT_FIELD ); ?>[notify_from]" value="<?php echo esc_attr( $options['notify_from'] ); ?>" />
+			<?php
+		}
+
+		/** Slack settings **/
+		public function sc_wpun_settings_slack_notifications_text() {
```

Comment:
Same

Reply:
this can not be removed, its required for the settings section